### PR TITLE
Restore `navigate-pages-with-arrow-keys`

### LINKED ISSUE #3466: `navigate-pages-with-arrow-keys` doesn't seem to work
1. Visit paginated view
2. Press right arrow key
3. Nothing happens

Paginated view examples:

https://github.com/sindresorhus/refined-github/pulls?q=
https://github.com/sindresorhus/refined-github/commits/master

### PR DESCRIPTION
**Pull request by artusm**
1. LINKED ISSUES:
Closes #3466

2. TEST URLS:
   [https://github.com/sindresorhus/refined-github/releases](https://github.com/sindresorhus/refined-github/releases)
	[https://github.com/sindresorhus/refined-github/pulls?q=](https://github.com/sindresorhus/refined-github/pulls?q=)
[https://github.com/sindresorhus/refined-github/commits/master](https://github.com/sindresorhus/refined-github/commits/master)
[https://github.com/sindresorhus/refined-github/search?q=+is%3Aissue&type=Issues](https://github.com/sindresorhus/refined-github/search?q=+is%3Aissue&type=Issues)
3. SCREENSHOT:

### CODE REVIEW

File: source/features/navigate-pages-with-arrow-keys.tsx [hunk 1]
```diff
@@ -2,13 +2,31 @@ import select from 'select-dom';
...
+	'a.previous_page',
+	/* Commits */
+	'.paginate-container > .BtnGroup .btn:first-child',
+	/* Releases */
+	'.paginate-container > .pagination > *:first-child'
+];
+
 function init(): void {
-	const createNextPageButton = select('a.next_page');
+	const createNextPageButton = select(nextPageButtonSelectors.join(' , '));
```
Comment (yakov116) [inline source/features/navigate-pages-with-arrow-keys.tsx:24]:
```suggestion
	const createNextPageButton = select(nextPageButtonSelectors.join());
```

File: source/features/navigate-pages-with-arrow-keys.tsx [hunk 2]
```diff
@@ -2,13 +2,31 @@ import select from 'select-dom';
 
 import features from '.';
 
+const nextPageButtonSelectors = [
+	/*  Issue/PR list, Search */
```
Comment (yakov116) [inline source/features/navigate-pages-with-arrow-keys.tsx:6]: Can you align the comment to be on the same line as the selector?

File: source/features/navigate-pages-with-arrow-keys.tsx [hunk 3]
```diff
@@ -2,13 +2,31 @@ import select from 'select-dom';
...
+
 function init(): void {
-	const createNextPageButton = select('a.next_page');
+	const createNextPageButton = select(nextPageButtonSelectors.join(' , '));
 	if (createNextPageButton) {
 		createNextPageButton.dataset.hotkey = 'ArrowRight';
 	}
 
-	const createPreviousPageButton = select('a.previous_page');
+	const createPreviousPageButton = select(previousPageButtonSelectors.join(' , '));
```
Comment (yakov116) [inline source/features/navigate-pages-with-arrow-keys.tsx:29]:
```suggestion
	const createPreviousPageButton = select(previousPageButtonSelectors.join());
```

File: source/features/navigate-pages-with-arrow-keys.tsx [hunk 4]
```diff
@@ -2,13 +2,25 @@ import select from 'select-dom';
 
 import features from '.';
 
+const nextPageButtonSelectors = [
+	'a.next_page', // Issue/PR list, Search
+	'.paginate-container > .BtnGroup .btn:last-child', // Commits
+	'.paginate-container > .pagination > :last-child' // Releases
```
Comment (fregante) [inline source/features/navigate-pages-with-arrow-keys.tsx:8]: Should `.btn` be dropped from both?
```suggestion
	'.paginate-container > .BtnGroup > :last-child', // Commits
	'.paginate-container > .pagination > :last-child' // Releases
```

**Pull request merged**